port/cpl_odbc.h

```diff
@@ -174,6 +174,7 @@ class CPL_DLL CPLODBCSession {
 
     CPL_DISALLOW_COPY_ASSIGN(CPLODBCSession)
 
+  protected:
```

Comment:
Can you add the following line just below protected
``//! @cond Doxygen_Suppress
``
and that one just before public:
``//! @endcond
``
to avoid Doxygen complaining about undocumented members.

Same for CPLODBCStatement class